modules/data.atmosphere/R/download.ECMWF.R

```diff
@@ -0,0 +1,201 @@
+#' Downloads ECMWF Open Data 15 day forecast data products 
+#' https://confluence.ecmwf.int/display/DAC/ECMWF+open+data%3A+real-time+forecasts
+#' 
+#' https://github.com/ecmwf/ecmwf-opendata
+#' Under the hood, this function uses the Python `ecmwf-opendata` module,
+#' which can be installed via `pip` (`pip install --user ecmwf-opendata`). The
+#' module is accessed via the `reticulate` package.
+#'
+#' @param outfolder location on disk where outputs will be stored
+#' @param lat.in,lon.in site coordinates, decimal degrees (numeric)
+#' @param fchour reference time of the forecasts. Values are 00, 12 for 15 day forecast.
+#' @param stream  forecasting system that produces the data. Current value is `"enfo"`
+#' @param type the type of forecast data. Current values: `cf` (controlled forecast), `pf` (perturbed forecast)
+#' @param parameters character vector of product types, or `"all"`.
+#' @param reticulate_python Path to Python binary for `reticulate`
+#'   (passed to [reticulate::use_python()]). If `NULL` (default), use
+#'   the system default.
+#' @param overwrite Logical. If `FALSE` (default), skip any files with
+#'   the same target name (i.e. same variable) that already exist in
+#'   `outfolder`. If `TRUE`, silently overwrite existing files.
+#' @return information about the output file
+#' @export
```

Comment:
Could you also run roxygen (package v7.1.2) so that the `NAMESPACE` will be updated